pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+This function uses Fisher's definition of kurtosis (kurtosis of normal
+== 0.0) without bias.
+
+Returns
+-------
+same type as input
```

Comment:
Again, I know this was already there ..:) But I also find this not very clear. What is the "input"? As I didn't provide anything to the function .. (it's rather the object calling rolling) 

But not directly sure how to document this very well. Maybe just "Series/DataFrame" (depending on the calling object)

Reply:
So when I did this originally I had:

```python
Series or DataFrame
    Like-indexed object containing the result of the kurt operation
```

However, I ended up with the above because all of the other rolling / expanding functions used this in the `_doc_template` variable (linked below). We could either break that consistency or update the `_doc_template` variable to reflect the above (perhaps substituting `kurt` for `%(name)`) if we think that's a better generally applicable description

https://github.com/pandas-dev/pandas/blob/aedbd948938f7e9230a321eb49f6c789867ab2b6/pandas/core/window.py#L51

Reply:
I think we should break the consistency for now, with the idea that if all those docstrings get updated during the sprint, we should strive again for consistency with the new wording?

Or indeed already update the template, then it is changed for all as well. 

In any case I think we should try to make the docstring that you are now editing, as good as possible.